homeassistant/components/matter/switch.py

```diff
@@ -61,6 +63,49 @@ def _update_from_device(self) -> None:
         )
 
 
+@dataclass(frozen=True)
+class MatterNumericSwitchEntityDescription(
+    SwitchEntityDescription, MatterEntityDescription
+):
+    """Describe Matter Numeric Switch entities."""
+
+
+class MatterNumericSwitch(MatterSwitch):
```

Comment:
Since we add a new entity class we need to test the service action calls and not only the state with a snapshot. See existing switch tests for examples.